expansion/src/main/java/no/unit/nva/expansion/model/ExpandedGeneralSupportRequest.java

```diff
@@ -0,0 +1,53 @@
+package no.unit.nva.expansion.model;
+
+import static no.unit.nva.expansion.model.ExpandedTicket.Constants.IDENTIFIER_FIELD;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import java.net.URI;
+import java.util.Set;
+import no.unit.nva.expansion.ResourceExpansionService;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.model.PublicationSummary;
+import no.unit.nva.publication.model.business.GeneralSupportRequest;
+import no.unit.nva.publication.service.impl.ResourceService;
+import nva.commons.apigateway.exceptions.NotFoundException;
+
+@JsonTypeName(ExpandedGeneralSupportRequest.TYPE)
+public class ExpandedGeneralSupportRequest implements ExpandedTicket {
```

Comment:
Currently the Expanded entry has insufficient data. After we have finished with the ticket service we should describe what we expect from an expanded ticket.